### PR TITLE
SW-6407 Added photo types for observation plot photos

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#2732** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2732?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2732?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt [hunk 1]
```diff
@@ -1227,6 +1230,8 @@ data class UpdatePlotObservationRequestPayload(
 data class UploadPlotPhotoRequestPayload(
     val gpsCoordinates: Point,
     val position: ObservationPlotPosition,
+    @Schema(description = "Type of observation plot photo. Defaulted to Plot")
```
Comment (sgrimm) [inline src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt]: FYI, `@Schema` has a `defaultValue` attribute you can use to document the default.

**Review by sgrimm — APPROVED** [review]
Looks solid!

**Comment by tommylau523** [comment]
### Merge activity

* **Dec 31, 5:33 PM EST**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/2732).

**Pull request merged**